Add wireframe.md, annotations.md, and term folders. (#5)
js6450/docs: content/uiux/concepts/user-workflows/user-workflows.md

---
Title: 'User Workflows' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
Description: 'User workflows are navigational paths throughout a digital interface.' # Required; ideally under 150 characters and starts with a present-tense verb (used in search engine results and content previews)
Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
  - 'Web Design'
Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
  - 'UI'
  - 'UX'
CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
  - 'intro-to-ui-ux'
  - 'path/front-end-engineer-career-path'
---

> User workflows are navigational paths throughout a digital interface. User workflows illustrate the path a user takes to complete a task, and there may be multiple workflows that different users take to complete the same task.

Reviewer: ```suggestion
User workflows are navigational paths throughout a digital interface. They illustrate the path a user takes to complete a task, and there may be multiple workflows that different users take to complete the same task.
```

![Information Architecture - Site Map example](https://static-assets.codecademy.com/Courses/intro-to-ui-and-ux/wireframes/information-architecture.svg)

> This image features a [site map](https://www.codecademy.com/resources/docs/uiux/site-map). In this site map, a path from homepage -> resources -> projects is considered a user workflow.

Reviewer: ```suggestion
This image features a [site map](https://www.codecademy.com/resources/docs/uiux/site-map). In this site map, the considered user workflow starts from the homepage, then resources, and lastly projects.
```
